### ISSUE TITLE
Cannot build with opam-installed extructures

### ISSUE THREAD

**Issue by emarzion** [arthuraa/coq-utils#2]
Relevant versions:
```
coq: 8.8.0
coq-extructures: 0.1.0
coq-mathcomp-algebra: 1.7.0
coq-mathcomp-fingroup: 1.7.0
coq-mathcomp-ssreflect: 1.7.0
```

Running make gives the following error:

```
File "./theories/nominal.v", line 2680, characters 24-35:
Error:
Ltac call to "by (ssrhintarg)" failed.
In environment
T : nominalType
A : {fset name}
x : T
sub : fsubset (A :&: names x)%fset (names x)
_s_ : {fperm name}
_Hyp_ : finsupp_perm fset0 _s_
_x_, _y_ : {fset name}
_Hyp1_ : nomR _s_ _x_ _y_
_x1_, _y1_ : T
_Hyp2_ : nomR _s_ _x1_ _y1_
Unable to unify "let (sort, _, _) := Nominal.eqType ?M1726 in sort" with
 "FSet.fset_type name_ordType".
```

After uninstalling `extructures` and reinstalling by hand everything works fine.

**Comment by arthuraa** [comment]
Thanks for the feedback. I made a change in the master branch of extensional structures and ported the change to Coq utils without leaving a note. I'll package the version of Coq utils that is compatible with extructures 0.1 to avoid future confusion.

**Comment by arthuraa** [comment]
OK, I just released version 0.1 on https://github.com/arthuraa/coq-utils/releases/tag/v0.1. This should compile with extructures v0.1, available on OPAM.
